php/WP_CLI/Formatter.php

```diff
@@ -347,21 +347,37 @@ private function assoc_array_to_rows( $fields ) {
 	}
 
 	/**
-	 * Transforms objects and arrays to JSON as necessary
+	 * Transforms into shell-friendly output, as necessary
 	 *
 	 * @param mixed $item
 	 * @return mixed
 	 */
-	public function transform_item_values_to_json( $item ) {
+	public function transform_item_values( $item ) {
```

Comment:
I don't think we can rename this public method, because of backward compatibility concerns. So let's undo this.

Reply:
Thanks for the feedback. Reverted it back to `transform_item_values_to_json`.